cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -549,6 +549,20 @@ libxml_disable_entity_loader(true);
 
 A description of how to abuse this in PHP is presented in a good [SensePost article](https://www.sensepost.com/blog/2014/revisting-xxe-and-abusing-protocols/) describing a cool PHP based XXE vulnerability that was fixed in Facebook.
 
+# Python
+The following information for XXE injection in python is directly from this [official python documentation page](https://docs.python.org/2/library/xml.html#xml-vulnerabilities).
```

Comment:
Given that Python2 will reach its end of life in about three months, I think it would be better to give the Python3 table (as that's why people should be using), with maybe a link to the Python2 table with a note that it's legacy and soon to be unsupported.

Also `Python` should be capitalised.

Reply:
I have added the main table reference for python 3 . And also link to python 2 with a note as suggested.